engine/core/src/main/java/org/datacleaner/util/CrosstabReducerHelper.java

```diff
@@ -1,114 +0,0 @@
-/**
- * DataCleaner (community edition)
- * Copyright (C) 2014 Neopost - Customer Information Management
- *
- * This copyrighted material is made available to anyone wishing to use, modify,
- * copy, or redistribute it subject to the terms and conditions of the GNU
- * Lesser General Public License, as published by the Free Software Foundation.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
- * or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License
- * for more details.
- *
- * You should have received a copy of the GNU Lesser General Public License
- * along with this distribution; if not, write to:
- * Free Software Foundation, Inc.
- * 51 Franklin Street, Fifth Floor
- * Boston, MA  02110-1301  USA
- */
-package org.datacleaner.util;
-
-import java.math.BigDecimal;
-import java.math.BigInteger;
-import java.util.Collection;
-import java.util.List;
-
-import org.datacleaner.result.Crosstab;
-import org.datacleaner.result.CrosstabDimension;
-import org.datacleaner.result.CrosstabNavigator;
-
-/**
- * Helper class for reductions of crosstabs
- *
- */
-public class CrosstabReducerHelper {
```

Comment:
Why was this moved from engine-core to basic-analyzers? Right now it seems to be shared between boolean analyzer and reference-data matcher. And potentially any analyzer that uses crosstabs. So shouldn't it be in a common place instead of in basic-analyzers which is a particular set of analyzers?